pkg/config/config.go

```diff
@@ -186,7 +191,7 @@ func (operatorConfig *NSXOperatorConfig) GetTokenProvider() auth.TokenProvider {
 }
 
 func (operatorConfig *NSXOperatorConfig) createTokenProvider() error {
-	log.V(1).Info("try to load VC host CA")
+	log.Info("try to load VC host CA")
```

Comment:
I guess they are debug log @TaoZou1

Reply:
fixed

Reply:
It's still confused why log.Info is used uere
also in func:
func (vcConfig *VCConfig) validate() error {
etc.

we suppose you use a different logger in this config.go, if that is case, why some functions are using log.Info, however, some are using log.Infof?

log.Info is supposed to be used in other modules, log.Infof is using sugar logger which is specific used in config.go. If that is the case, we had better use a newname for logger in config.go,
like, configLog. It will be easier to distinguish loggers used in different modules.

Reply:
OK, renamed.